truss/templates/server/common/truss_server.py

```diff
@@ -37,6 +37,7 @@
 DEFAULT_NUM_SERVER_PROCESSES = 1
 WORKER_TERMINATION_TIMEOUT_SECS = 120.0
 WORKER_TERMINATION_CHECK_INTERVAL_SECS = 0.5
+ACCESS_LOGGING_IGNORE_PATHS = ["/", "/v1/models/model"]
```

Comment:
maybe a comment indicating that these are healthchecks and we don't have access logging for these